main.c

```diff
@@ -157,275 +150,10 @@ static int open_handlers(void)
 	return num_good;
 }
 
-static void thread_cleanup(void *arg)
-{
-	struct tcmu_device *dev = arg;
-	struct tcmulib_handler *handler = tcmu_get_dev_handler(dev);
-	struct tcmur_handler *r_handler = handler->hm_private;
-
-	r_handler->close(dev);
-	free(dev);
-}
-
-static int generic_handle_cmd(struct tcmu_device *dev,
-			      struct tcmulib_cmd *tcmulib_cmd)
```

Comment:
Just a FYI, I busted this part of the patch with the WRITE_VERIFY PR merge. It added a new command to the switch below.